custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt

```diff
@@ -62,6 +62,26 @@ class CloudIssuePage(
         return CloudIssueLinking(driver)
     }
 
+    override fun changeAssignee(): CloudIssuePage {
+        driver
+            .findElement(By.cssSelector("[data-test-id = 'issue.views.field.user.assignee']"))
+            .click();
+
+        var firstUser = driver
+            .wait(
+                ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@id=\"react-select-assignee-option-0\"]"))
+            )
+
+        var userName = firstUser.text.removeSuffix(" (Assign to me)")
+        firstUser.click()
+
+        driver.wait(
+            ExpectedConditions.presenceOfElementLocated(By.ByXPath(String.format("//*[@data-test-id=\"issue.views.field.user.assignee\"]//*[.=\"%s\"]", userName)))
```

Comment:
We can improve readability by:
* using `'` instead of `\"` to surround XPath string literals
* using [Kotlin String templates](https://kotlinlang.org/docs/reference/basic-types.html#string-templates) rather than `String.format`
* using `By` static factory methods rather than class constructors
```suggestion
            ExpectedConditions.presenceOfElementLocated(By.xpath("//*[@data-test-id='issue.views.field.user.assignee']//*[.='$userName']"))
```